components/script/dom/element.rs

```diff
@@ -2353,6 +2353,7 @@ impl ElementMethods for Element {
                     rect.origin.y.to_f64_px(),
                     rect.size.width.to_f64_px(),
                     rect.size.height.to_f64_px(),
+                    CanGc::note(),
```

Comment:
If you modify Bindings.conf like https://github.com/servo/servo/blob/028026bebe992431ede7c57b7e3065616d859026/components/script/dom/bindings/codegen/Bindings.conf#L43, we can add CanGc arguments to the generated trait methods that you touch in this PR as well.

Reply:
sounds great! let me get on that :)